code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -452,6 +452,20 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		if(3) icon = 'icons/obj/pda_old.dmi'
 		if(4) icon = 'icons/obj/pda_rugged.dmi'
 		if(5) icon = 'icons/obj/pda_holo.dmi'
+		if(6)
+			icon = 'icons/obj/pda_wrist.dmi'
+			item_state = icon_state
+			item_icons = list(
+    			slot_belt_str = 'icons/mob/pda_wrist.dmi',
+    			slot_wear_id_str = 'icons/mob/pda_wrist.dmi',

```

Comment:
Took me a moment to realize these were in a list declaration, would you please indent the list's contents so they're clearly distinct from the surrounding variable definitions and it doesn't look like you're assigning a variable whose name is a #define phrase?